<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>204</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="unsupportedLayersLabel">
     <property name="visible">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QTableWidget" name="layersTable"/>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QgsFilterLineEdit" name="textFilterBox">
       <property name="placeholderText">
        <string>Filter layers...</string>
       </property>
       <property name="showSearchIcon">
        <bool>true</bool>
       </property>
       <property name="qgisRelation" stdset="0">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="showVisibleLayersOnlyCheckbox">
       <property name="text">
        <string>Show Visible Layers Only</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="multipleToggleButton">
       <property name="toolTip">
        <string>Toggle layers</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../resources/visibility.svg</normaloff>../resources/visibility.svg</iconset>
       </property>
       <property name="popupMode">
        <enum>QToolButton::InstantPopup</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="settingsPackagingButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>Set packaging layers action</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="popupMode">
        <enum>QToolButton::InstantPopup</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFilterLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qgsfilterlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
